index.html

```diff
@@ -422,8 +447,10 @@ <h3>Why us</h3>
                             <li>
                                 <div class="block_content">
                                     <div class="text_content">
-                                        <p class="title_block">Title</p>
-                                        <p class="text_block">Tur solvendae ego archetypi hoc attentius. At potestque ii extitisse in ingeniosi contingit. Partibus vis diversas pauciora actualis vox imo agi. Efficta nec quaedam capacem ita invenit hae cum. Illis mundo aliis ei ab magna. Illi ut quod si sese esto. Is pauciora im ad possumus absoluta pendeant compages. </p>
+                                        <p class="title_block">Quality</p>
+                                        <p class="text_block">
+                                        We always aim for the best player experiences.
```

Comment:
We always aim for the best player experiences. And we provides support for operation